ingest/settings.py

```diff
@@ -0,0 +1,64 @@
+# File is responsible for defining globals and initializing them
+from mongo_connection import MongoConnection
+from bson.objectid import ObjectId
+
+MONGO_CONNECTION = MongoConnection()
+
+
+def init(study_id, study_file_id):
```

Comment:
Good stuff.  I did some research and realized why we were talking past each other so much on Monday.  'global' in python is actually module-scoped, which is what I want.  I didn't realize that and thought when you were saying these were global, that they were not module-scoped.  so we're on the same page -- I just didn't know the pythonese  

I believe best practice is to name variables like this with double-underscores. e.g.   __study__  to indicate that other modules are not supposed to access this variable directly, and instead go through your getters

Reply:
I believe we paired on this. I refactored this setup. I isolated [study](https://github.com/broadinstitute/scp-ingest-pipeline/blob/626b12150350eec3a08e60b76202418e69dc701f/ingest/config.py#L58) and[ study files](https://github.com/broadinstitute/scp-ingest-pipeline/blob/626b12150350eec3a08e60b76202418e69dc701f/ingest/config.py#L89) to have their own classes where each class has access to their respective documents. I used the ```@property``` decorator so that they have getter and setter methods.